### PR TITLE
Add grpc metrics, all of them, and fix remote logging, no more errors

### PR DESCRIPTION
**Pull request by pcarranza**
Nothing more, nothing less

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -129,6 +130,8 @@ func launch(args args) (func(), error) {
 		}, nil
 
 	case "agent":
+		// metrics.RegisterAgentMetrics()
```
Comment (omame) [inline main.go]: What is this about?

Comment (pcarranza) [reply main.go]: I doesn't exists yet, IIRC

Comment (pcarranza) [reply main.go]: It's there as a placeholder to remind me to do it, at some point.

Comment (omame) [reply main.go]: Gotcha.

**Pull request merged**